src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelectorCore.js

```diff
@@ -31,11 +31,18 @@ const DefaultSelectorRow = ({ item = {}, ...rest }: DefaultSelectorRowProps) =>
 
 const DefaultStartMentionMessage = () => <FormattedMessage {...messages.startMention} />;
 
-type MentionStartStateProps = {
+type MentionStateProps = {
+    contacts?: Number,
     message?: React.Node,
 };
 
-const MentionStartState = ({ message }: MentionStartStateProps) => <div className="mention-start-state">{message}</div>;
+const MentionStartState = ({ message }: MentionStateProps) => <div className="mention-start-state">{message}</div>;
```

Comment:
We typically like to keep a single component per file. That being said, do we even need `MentionStartState` as a separate component? Seems like that could be inlined on L303. Similarly for the `MentionUsersFound`